src/stock-status-ribbon.html

```diff
@@ -11,7 +11,7 @@
     >
       <span>
         <span
-          ng-bind="$ctrl.data[bar.id].value | number: 0"
+          ng-bind="$ctrl.data[bar.id].value | number: ($ctrl.config.decimalPlaces || 0)"
```

Comment:
Yeah, I also thought about that, just did it like this because the component doesn't define a controller and I took the lazy option of just modifying it on the view. But your comment made me feel ashamed of my lazyness! I'll change it 😁